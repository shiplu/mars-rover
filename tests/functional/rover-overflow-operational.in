5 3
1 1 N
R
1 2 N
R
1 3 N
R
2 1 N
R
2 2 N
R
2 3 N
R
3 1 N
R
3 2 N
R
3 3 N
R
4 1 N
L
4 2 N
L
4 3 N
L
5 1 N
RR
5 2 N
RR
5 3 N
RR
