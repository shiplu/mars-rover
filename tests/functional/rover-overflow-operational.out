1 1 E OPERATIONAL
1 2 E OPERATIONAL
1 3 E OPERATIONAL
2 1 E OPERATIONAL
2 2 E OPERATIONAL
2 3 E OPERATIONAL
3 1 E OPERATIONAL
3 2 E OPERATIONAL
3 3 E OPERATIONAL
4 1 W OPERATIONAL
4 2 W OPERATIONAL
4 3 W OPERATIONAL
5 1 S OPERATIONAL
5 2 S OPERATIONAL
5 3 S OPERATIONAL
